### PR TITLE
Bah 8940 hospitalization update

### PR DESCRIPTION
**Pull request by imeecuison**
## Description
Update to hospitalization date entry for 8940.
NOTE: This PR will go through Internal Review but then will have to wait until the associated vets-json-schema PR has been merged into master before going through external review.

http://localhost:3001/disability-benefits/apply/form-526-all-claims/hospitalization-history
## Testing done
tests/pages/hospitilizationHistory.unit.spec.jsx

## Screenshots

screen shot 2019-01-04 at 3 39 26 pm
screen shot 2019-01-04 at 3 39 19 pm

## Acceptance criteria
- [ ] - Content has changed to the following:
 Please tell us where and when you spent time in a hospital.

  Hospital’s name { text field }

  Hospital’s address { address pattern }

  When were you in this hospital? Include the dates you were admitted and discharged for each 
  stay. { text box }

  Add Another { button }

  If you received care from a private medical facility, we can get that information for you. We'll ask 
  you about that later in this process.

  If you received care at a VA medical facility, we have access to those records and can get them 
  for you.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/content/unemployabilityFormIntro.jsx [hunk 1]
```diff
@@ -6,6 +6,12 @@ export const unemployabilityTitle = (
   </legend>
 );
 
+export const unemployabilityTitleHospitalization = (
+  <legend className="schemaform-block-title schemaform-title-underline">
+    Individual Unemployability: Hospitalization
```
[the remark this answers is not in the document]
Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/content/unemployabilityFormIntro.jsx]: Use `import { unemployabilityTitle } from '../content/unemployabilityFormIntro';` for the main title, See https://github.com/department-of-veterans-affairs/vets-website/blob/bah-8940-doctor-interview/src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js [hunk 2]
```diff
@@ -1,32 +1,47 @@
-import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import React from 'react';
```
Comment (zurbergram) [inline src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: file extension should be `.jsx` if using React

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js [hunk 3]
```diff
@@ -1,32 +1,47 @@
...
 
 import HospitalizationPeriodView from '../components/HospitalizationPeriodView';
 
-import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
-import { unemployabilityTitle } from '../content/unemployabilityFormIntro';
+// import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
+import { unemployabilityTitleHospitalization } from '../content/unemployabilityFormIntro';
```
Comment (zurbergram) [inline src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: Use `import { unemployabilityTitle } from '../content/unemployabilityFormIntro';` for the main title, See https://github.com/department-of-veterans-affairs/vets-website/blob/bah-8940-doctor-interview/src/applications/disability-benefits/all-claims/pages/unemployabilityCertification.js

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js [hunk 4]
```diff
@@ -1,32 +1,47 @@
...
+
+import tempSchema from '../config/schema'; // Will change to fullSchema when json schema approved
+
+const { hospitalizationHistory } = tempSchema.properties;
+
+const recordsDescription = (
+  <div>
+    <p>
+      If you received care from a private medical facility, we can get that
+      information for you. We'll ask you about that later in this process.
```
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: fixed! thank you!

File: src/applications/disability-benefits/all-claims/content/unemployabilityFormIntro.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/content/unemployabilityFormIntro.jsx]: changed back to old way!

[the remark this answers is not in the document]
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/content/unemployabilityFormIntro.jsx]: screen shot 2019-01-08 at 10 31 36 am

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js (hunk 2 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: changed!

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js (hunk 3 above)
Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.js]: changed!

File: src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.jsx [hunk 5]
```diff
@@ -1,32 +1,49 @@
...
-        ),
         hospitalName: {
           'ui:title': 'Name of hospital',
         },
+        hospitalizationDateRange: {
+          'ui:title':
+            'When were you in this hospital? Include the dates you were admitted and discharged for each stay.',
+        },
         hospitalAddress: {
           addressLine1: {
```
Comment (zurbergram) [inline src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.jsx]: should this use the `addressUI` ?

Comment (imeecuison) [reply src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.jsx]: i ran into a problem where addressUI uses street1 and street2. and I needed addressLine1 and addressLine2

Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/pages/hospitalizationHistory.jsx]: Ok, i'll fix this in my 8940 schema changes for vets-website

**Review by zurbergram — APPROVED** [review]
LGTM

**Comment by zurbergram** [comment]
This work will be resolved in https://github.com/department-of-veterans-affairs/vets-website/pull/9361

**Pull request closed without merging**